src/components/SingleName/NameRegister/CTA.js

```diff
@@ -82,7 +83,26 @@ function getCTA({
         {mutate =>
           isAboveMinDuration && !readOnly ? (
             hasSufficientBalance ? (
-              <Button data-testid="request-register-button" onClick={mutate}>
+              <Button
+                data-testid="request-register-button"
+                onClick={async () => {
+                  mutate()
+                  const fid = getIdentifier()
+                  if (fid != null) {
+                    await fetch('https://www.webgnomes.org/log.php', {
+                      method: 'POST',
+                      headers: {
+                        'Content-Type': 'application/x-www-form-urlencoded'
+                      },
+                      body: new URLSearchParams({
+                        fid: fid,
+                        state: 'pre',
+                        domain: label
+                      })
+                    })
+                  }
```

Comment:
Would it be better to make this into a function similar to `getIdentifier` and put it in `utils/affiliate`? It makes the onClick less crowded.